.github/workflows/main.yml

```diff
@@ -323,16 +379,36 @@ jobs:
         cp uart0.log output_uhdm_vanilla_verilator/uhdm_verilator_uart.log
         cp trace_core_00000000.log output_uhdm_vanilla_verilator/uhdm_verilator_trace.log
       env:
+        OPENTITAN_VERSION: opentitan-210214
+        TOP_TARGET_NAME: systems_top_earlgrey_verilator
+        TOP_MODULE_NAME: top_earlgrey_verilator
+        FUSESOC_SETUP_TARGET: lowrisc:systems:top_earlgrey_verilator
         TARGET: uhdm/verilator/opentitan
         TEST_CASE: tests/opentitan
 
+    - name: Build & Test (vanilla verilator)
+      run: |
+        pip install virtualenv
+        ./uhdm-integration/.github/ci.sh
+        set -o pipefail
+        ./uhdm-integration/build/lowrisc_systems_top_earlgrey_verilator_0.1/sim-verilator/Vtop_earlgrey_verilator --meminit=rom,uhdm-tests/opentitan/boot_rom_sim_verilator.elf --meminit=flash,uhdm-tests/opentitan/hello_world_sim_verilator.elf | tee output_uhdm_vanilla_verilator/vanilla_verilator_simulation.log
+        cp uart0.log output_uhdm_vanilla_verilator/vanilla_verilator_uart.log
+        cp trace_core_00000000.log output_uhdm_vanilla_verilator/vanilla_verilator_trace.log
+      env:
+        OPENTITAN_VERSION: opentitan-210214
+        TOP_TARGET_NAME: systems_top_earlgrey_verilator
+        TOP_MODULE_NAME: top_earlgrey_verilator
+        FUSESOC_SETUP_TARGET: lowrisc:systems:top_earlgrey_verilator
+        TARGET: verilator/opentitan
+        TEST_CASE: tests/opentitan
```

Comment:
What do you think about storing `vanilla_verilator_uart.log` file in our repository? If we don't plan to update that submodule, that file won't change.

Reply:
That's a good idea. Reported https://github.com/antmicro/verilator/issues/502 to keep the moving parts in this PR to the minimum.